ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.16__multi-nightly-4.16-upgrade-from-stable-4.15.yaml

```diff
@@ -333,8 +333,8 @@ tests:
     test:
     - chain: openshift-upgrade-qe-test
     workflow: cucushift-installer-rehearse-azure-ipi-cco-manual-workload-identity
-- as: azure-ipi-ovn-ipsec-arm-mixarch-f28
-  cron: 21 21 26 * *
+- as: azure-ipi-ovn-ipsec-arm-mixarch-day2-f28
```

Comment:
No such job in `openshift-openshift-tests-private-release-4.15__multi-nightly.yaml`

Reply:
https://github.com/openshift/release/pull/55329/files#r1708727047